app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -0,0 +1,102 @@
+package protect.card_locker
+
+import android.os.Bundle
+import android.view.MenuItem
+import android.view.View
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import protect.card_locker.databinding.AboutActivityBinding
+
+class AboutActivity : CatimaAppCompatActivity() {
+    private var binding: AboutActivityBinding? = null
+    private var content: AboutContent? = null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val binding = AboutActivityBinding.inflate(layoutInflater)
+            .also { this.binding = it }
+        val content = AboutContent(this)
+            .also { this.content = it }
+
+        title = content.pageTitle
+
+        setContentView(binding.root)
+        setSupportActionBar(binding.toolbar)
+        enableToolbarBackButton()
+
+        val copyright = binding.creditsSub
+        copyright.text = content.copyright
+        val versionHistory = binding.versionHistorySub
+        versionHistory.text = content.versionHistory
+
+        binding.versionHistory.tag = "https://catima.app/changelog/"
+        binding.translate.tag = "https://hosted.weblate.org/engage/catima/"
+        binding.license.tag = "https://github.com/CatimaLoyalty/Android/blob/master/LICENSE"
+        binding.repo.tag = "https://github.com/CatimaLoyalty/Android/"
+        binding.privacy.tag = "https://catima.app/privacy-policy/"
+        binding.reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
+        binding.rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"
+
+        bindClickListeners()
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        val id = item.itemId
+        if (id == android.R.id.home) {
+            finish()
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        content?.destroy()
+        content = null
+        clearClickListeners()
+        binding = null
+    }
+
+    private fun bindClickListeners() {
+        val binding = binding ?: return
```

Comment:
I'm not a fan of this. If binding is somehow null, we're just silently not making anything clickable. This means the app doesn't work and also won't crash, meaning I won't be able to see the issue in the Google Play Console crashes meaning the app is broken for users without me ever knowing (as almost nobody ever reports a bug they run into).

Reply:
I understand your point. 

The whole reason behind this is, that in kotlin we have nullability and you have to deal with it.
And in my opinion it looks better, then `val binding = binding!!`. https://kotlinlang.org/docs/null-safety.html#the-operator

In all this cases it depend on the lifecycle and that would mean that something is completely messed up, if one of those is null.

Reply:
I understand that something is completely messed up if this case ever happens, but I strongly prefer the result of something being completely messed up causing a crash and thus it getting logged in the Google Play Developer Console instead of the app misbehaving without me ever knowing. If I somehow ever manage to accidentally turn the binding variable into null due to bad code, I would be far more likely to miss it in testing if the code hides these errors by trying to somewhat recover from them and accidentally push such a bug into production.